lib/bloomFilterReader.js

```diff
@@ -0,0 +1,128 @@
+const parquet_util = require("./util");
```

Comment:
I think we need to organize this a little better...not sure yet.

bloomFilter/
  util/
    sbbf.js
  - BloomFilterReader.js
  -  BloomFilterWriter.js
  - bloomFilter.js